### PR TITLE
Improve build

### PR DESCRIPTION
**Pull request by adamretter**
This switches the build from Ant to Maven and also adds DocBook validation to the documentation as part of the build. This should ensure that we cannot create/publish an invalid Documentation App (which previously happened!!!).

### CODE REVIEW

**Comment by adamretter** [comment]
@duncdrum Hopefully this should fix the issue (https://github.com/eXist-db/documentation/issues/134) that you reported from occurring again

**Comment by adamretter** [comment]
@joewiz That issue should be addressed now.

**Comment by adamretter** [comment]
@duncdrum I don't see why not :-)

File: README.md [hunk 1]
```diff
@@ -45,4 +46,4 @@ $ mvn release:perform
 
 3. An EXPath Application Package (.xar file) is deposited in the `target` directory
 
-4. If you are a core contributor, you should then commit and pish.
\ No newline at end of file
+4. If you are a core contributor, you should then commit and pish.
```
Comment (joewiz) [inline README.md]: Typo alert! That's `push` ;)

File: pom.xml [hunk 2]
```diff
@@ -90,17 +90,46 @@
...
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <validationSets>
+                        <validationSet>
+                            <dir>src/main/xar-resources/data</dir>
+                            <excludes>
+                                <exclude>*.xq</exclude>
+                                <exclide>xinclude.xml</exclide>
```
Comment (joewiz) [inline pom.xml]: Typo alert! `<exclide>` should be `<exclude>`.

**Comment by joewiz** [comment]
@adamretter The travis integration is very cool! Is the xar building step included in the test just to be sure there aren't errors when zipping up the files?

File: README.md (hunk 1 above)
Comment (adamretter) [reply README.md]: Ah ha! I have fixed that now.

File: pom.xml (hunk 2 above)
Comment (adamretter) [reply pom.xml]: Thanks! I have fixed that now.

**Comment by adamretter** [comment]
@joewiz Exactly, it ensures that the build completes.

In future, we could go further, by having a Maven plugin which starts-up eXist-db deploys the XAR into eXist, and checks a few URLs.

**Comment by joewiz** [comment]
@adamretter Thank you!

**Pull request merged**